msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalAuthResult.java

```diff
@@ -42,4 +45,20 @@ public MsalAuthResult(@NonNull final IAuthenticationResult authenticationResult)
     public MsalAuthResult(@NonNull final Exception exception) {
         super(exception);
     }
+
+    public String verifyATForPop(@NonNull final String rawIdToken) throws ServiceException {
```

Comment:
Okay, you're verifying accessToken (NOT IdToken); and I don't think you need to even accept AT as a param here...since you have access to the AT in this class itself..you can just invoke the `getAccessToken` method directly in this method to get the AT

Reply:
we do not have access to SHR in this class. This method would be verifying both the authResult.getAccessToken and SHR

Reply:
Okay sounds good. In that case then you can keep as is, however, just make 2 changes:

- Make this method a `static` method as opposed to `instance` method
- Rename the param from `rawIdToken` to `shr`

and then it should be good to be used for both AT / SHR. 

Also feel free to add the dependency for assertions to the automation app

Reply:
added the dependancy and used assertions